src/main/kotlin/main/main.kt

```diff
@@ -18,9 +18,9 @@ fun main(args: Array<String>) {
     val notary = NotaryInitialization()
 
     IrohaInitializtion.loadIrohaLibrary()
-            .flatMap { notary.init() }
-            .failure {
-                logger.logger.error { it }
-                System.exit(1)
-            }
+        .flatMap { notary.init() }
```

Comment:
How do you format the code?

Reply:
option + cmd + L

Reply:
IntelliJ Idea settings are described here (https://kotlinlang.org/docs/reference/coding-conventions.html#applying-the-style-guide)

Reply:
The point is that I format it the same way, and I get different result :D